src/app/containers/ProfilePageWrapper/index.tsx

```diff
@@ -1,20 +1,70 @@
 import classNames from "classnames/bind";
 import { Typography, Pagination, LinearProgress } from "@mui/material";
-import { useState, useCallback, useMemo } from "react";
+import Tabs from "@mui/material/Tabs";

```

Comment:
Is it possible to merge them together? => import { Typography, Pagination, LinearProgress, Tabs, Tab, Box } from "@mui/material";